janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/ElasticSearchScrollTest.java

```diff
@@ -0,0 +1,130 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.diskstorage.es;
+
+import io.vavr.collection.List;
+import org.janusgraph.diskstorage.indexing.RawQuery;
+import org.junit.Test;
```

Comment:
nit: this should be the junit5 test-method.

Reply:
fixed

Reply:
done